README.md

```diff
@@ -292,28 +292,7 @@ Amongst the many awesome contributors, there are the maintainers. These maintain
 * Committing to reviewing pull requests, issues, and addressing comments/questions as quickly as possible
 * Acting as a point of contact for questions
 
-Just tag **@intelsdi-x/snap-maintainers** if you need to get some attention on an issue. If at any time, you don't get a quick enough response, reach out to any of the following team members directly:
```

Comment:
@IzabellaRaulin I was asked by @lynxbat to remove it since it consistently gets stale. We favor of pointing people to Slack. I'd be game to reintroduce it when we generate the table automatically from members of the `@intelsdi-x/snap-maintainers` team.